zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Content.java

```diff
@@ -417,19 +410,19 @@ boolean moveFile(String fromUri, String toUri) throws IOException {
         return false;
     }
 
-    static boolean isVisibleForCollectionOwner(CollectionOwner collectionOwner, Path entry) {
-        if (collectionOwner.equals(CollectionOwner.DATA_VISUALISATION)) {
-            return Files.isDirectory(entry)
-                    && isDataVisualisation(entry)
-                    && !isTimeseries(entry)
-                    && isNotPreviousVersions(entry);
-        } else {
-            // PUBLISHING SUPPORT
-            return Files.isDirectory(entry)
-                    && !isTimeseries(entry)
-                    && isNotPreviousVersions(entry)
-                    && !isDataVisualisation(entry);
+    static boolean isVisible(Path entry) {
+        return Files.isDirectory(entry)
+                && !isTimeseries(entry)
+                && isNotPreviousVersions(entry)
+                && !isDataVisSubDir(entry);
```

Comment:
Don't we want to show the data vis directories now? this is preventing them from being shown?